apps/feedahelp/src/components/ui/features/hero/Hero.tsx

```diff
@@ -60,9 +60,9 @@ const Hero = () => {
 
     return () => {
       window.removeEventListener("resize", handleResize);
-      clearInterval(autoScrollRef.current as any);
+      clearInterval(autoScrollRef.current as NodeJS.Timeout);
     };
-  }, []);
+  });
```

Comment:
We put an empty [], if we want the code inside useEffect to run only once. Without empty [], the code inside the useEffect will run on every render

Reply:
done.